assets/templates/index.gohtml

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./assets/static/css/style.css">
+  <title>{{.PageTitle}}</title>
+</head>
+
+<body>
+  {{ $context := . }}
+  <div class="container">
+    <h1>{{.PageTitle}}</h1>
+    <form action="/popup" method="post">
+      <!-- User -->
+      <div class="row">
+        <div class="col-25">
+          <label for="user">{{.Content.UserLabel}}</label>
```

Comment:
Prefer spaced for interpolation - it helps with readability:

```suggestion
          <label for="user">{{ .Content.UserLabel }}</label>
```

(throughout)